cmake/FindFreeImage.cmake

```diff
@@ -0,0 +1,55 @@
+#
+# Try to find the FreeImage library and include path.
+# Once done this will define
+#
+# FREEIMAGE_FOUND
+# FREEIMAGE_INCLUDE_PATH
+# FREEIMAGE_LIBRARY
+#
+
+IF (WIN32)
+	FIND_PATH( FREEIMAGE_INCLUDE_PATH FreeImage.h
```

Comment:
Could you replace the tabs in this file with 4 spaces?

Reply:
Done

`expand -t 4 FindFreeImage.cmake > FindFreeImage.cmake-tmp && mv FindFreeImage.cmake-tmp FindFreeImage.cmake`